specs/core/1_shard-data-chains.md

```diff
@@ -75,207 +91,191 @@ The following types are defined, mapping into `DomainType` (little endian):
 
 | Name | Value |
 | - | - |
-| `PLACEHOLDER` | `2**32` |
+| `PLACEHOLDER` | `2**3` |
```

Comment:
```suggestion
```

Reply:
And the whole `### TODO PLACEHOLDER` section.